api/Controllers/AuthController.cs

```diff
@@ -36,11 +37,31 @@ public IActionResult Login([FromBody]Login request)
             return Ok(token);
         }
 
-        private Account GetUserAccount(string email, string password)
+        [Route("registration")]
+        [HttpPost]
+        [Authorize]
+        public IActionResult CreateAccount([FromBody] Login request)
         {
-            return _hotelsDb.Accounts.SingleOrDefault(u => u.Email == email && u.Password == password);
+            var user = new Account
+            {
+                Email = request.Email,
+                Password = request.Password
+            };
+            if (!IsEmailUnique(user.Email))
+            {
+                return BadRequest("Is already registered with this email");
+            }
```

Comment:
Do I need to create a user before checking email uniqueness? If the email is not unique, then memory allocation was useless?

Reply:
Is used to check if token is expired

Reply:
Genial, why only I didn't think about this point myself